Frends.MicrosoftDynamics365.Request/Frends.MicrosoftDynamics365.Request/Frends.MicrosoftDynamics365.Request.cs

```diff
@@ -94,7 +94,7 @@ private static async Task<JToken> ExecuteRequestAsync(Input input, Options optio
         }
 
         var jsonResponse = await response.Content.ReadAsStringAsync(cancellationToken);
-        var contacts = JToken.Parse(jsonResponse);
+        var contacts = !string.IsNullOrWhiteSpace(jsonResponse) ? JToken.Parse(jsonResponse) : JToken.Parse("{\r\n\"message\": \"Success\"\r\n}");
```

Comment:
Can we actually rename this to something better that "contacts". I know that this is not from this issue, but just in general this would be more appropriate to be called like `response` or something.